app/src/main/java/de/ur/explure/views/CreatedRoutesFragment.kt

```diff
@@ -0,0 +1,83 @@
+package de.ur.explure.views
+
+import android.app.AlertDialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import de.ur.explure.R
+import de.ur.explure.adapter.RouteAdapter
+import de.ur.explure.model.route.Route
+import de.ur.explure.viewmodel.CreatedRoutesFragmentViewModel
+import kotlinx.android.synthetic.main.fragment_created_routes.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class CreatedRoutesFragment : Fragment(R.layout.fragment_created_routes),
+        RouteAdapter.OnItemClickListener, RouteAdapter.OnItemLongClickListener {
+
+    private val viewModel: CreatedRoutesFragmentViewModel by viewModel()
+    private lateinit var adapter: RouteAdapter
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        adapter = RouteAdapter(this, this)
```

Comment:
Vllt kannst du die ganze Adapter Initialisierung in eine extra `initializeAdapter()` Methode packen